coffee-shop/src/genericComponents/Button/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import styles from "../Button/Button.module.css";
+
+const Button = ({ label, handleClick }) => {
+  return (
+    <div className={styles.container}>
+      <button onClick={handleClick} className={styles.button}>
+        {label.text}
```

Comment:
you need to use text or label, and I recommend label, `label.text` will return error